cli.js

```diff
@@ -199,16 +199,18 @@ function statistics(objects) {
     countAmount = 0,
     countDate = 0;
   for (var key in objects) {
-    var object = objects[key];
-    if (!object.error) {
-      if (object.amount && object.date) {
-        ++countTotal;
-      }
-      if (object.date) {
-        ++countDate;
-      }
-      if (object.amount) {
-        ++countAmount;
+    if ({}.hasOwnProperty.call(key, objects)){
```

Comment:
I don't see a problem when we basically use it as a hash? We don't really care if there's inherited values.